{
    "comments": [
        {
            "author": "Michael McCandless",
            "body": "Thanks Jayendra!",
            "date": "2010-07-30T18:40:08.736+0000",
            "id": 0
        },
        {
            "author": "Robert Muir",
            "body": "reopening for possible 2.9.4/3.0.3 backport.\n",
            "date": "2010-10-29T13:03:14.601+0000",
            "id": 1
        },
        {
            "author": "Robert Muir",
            "body": "The bug doesn't exist in 3.0.x/2.9.x, it was introduced in LUCENE-2410 (in 3x and trunk)",
            "date": "2010-10-29T17:24:30.310+0000",
            "id": 2
        },
        {
            "author": "Grant Ingersoll",
            "body": "Bulk close for 3.1",
            "date": "2011-03-30T15:50:03.862+0000",
            "id": 3
        }
    ],
    "component": "",
    "description": "See thread \"MultiPhraseQuery throws ArrayIndexOutOfBounds Exception\" on dev@ by Jayendra Patil.",
    "hasPatch": false,
    "hasScreenshot": false,
    "id": "LUCENE-2580",
    "issuetypeClassified": "BUG",
    "issuetypeTracker": "BUG",
    "priority": "Major",
    "product": "LUCENE",
    "project": "LUCENE",
    "summary": "MultiPhraseQuery throws AIOOBE",
    "systemSpecification": true,
    "version": ""
}